server/test/read_authzmodel.go

```diff
@@ -2,18 +2,96 @@ package test
 
 import (
 	"context"
-	"errors"
 	"testing"
 
 	"github.com/openfga/openfga/pkg/id"
 	"github.com/openfga/openfga/pkg/logger"
+	"github.com/openfga/openfga/pkg/typesystem"
 	"github.com/openfga/openfga/server/commands"
 	serverErrors "github.com/openfga/openfga/server/errors"
 	"github.com/openfga/openfga/storage"
 	"github.com/stretchr/testify/require"
 	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
 )
 
+func TestSuccessfulReadAuthorizationModelQuery(t *testing.T, datastore storage.OpenFGADatastore) {
+	var tests = []struct {
+		name            string
+		storeID         string
+		modelID         string
+		schemaVersion   typesystem.SchemaVersion
+		typeDefinitions []*openfgapb.TypeDefinition
+	}{
+		{
+			name:          "write and read a 1.0 model",
+			storeID:       id.Must(id.New()).String(),
+			modelID:       id.Must(id.New()).String(),
+			schemaVersion: typesystem.SchemaVersion1_0,
+			typeDefinitions: []*openfgapb.TypeDefinition{
+				{
+					Type: "user",
+				},
+				{
+					Type: "document",
+					Relations: map[string]*openfgapb.Userset{
+						"reader": {
+							Userset: &openfgapb.Userset_This{},
+						},
+					},
+				},
+			},
+		},
+		{
+			name:          "write and read an empty 1.1 model",
```

Comment:
What do we mean by "empty 1.1 model"? Should this just be named "write and read a 1.1 model"?

Reply:
Yes, that is a typo. Thanks. Fixed.